src/app/screens/Home/components/Selector/styles.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../../shared/shared";
+
+.select-filter select {
+  background: $dim-grey;
+  height: 11px;
+  width: 15px;
+}
+
+.selector {
+  margin-right: 2.5%;
+}
```

Comment:
Styles should be wrapped in a single object. ask @juanisierra how to to this and why 😄